tools.yml

```diff
@@ -0,0 +1,65 @@
+install_tool_dependencies: True
```

Comment:
@bgruening This file shouldn't have been added, right?

This is generated by `planemo training_init`, probably should be created as a temporary file.

Reply:
Sorry, it was created when I ran planemo but I should not have added it.